src/jemalloc.c

```diff
@@ -104,6 +104,22 @@ const char *zero_realloc_mode_names[] = {
 	"abort",
 };
 
+prof_time_resolution_t opt_prof_time_resolution =
+	prof_time_resolution_default;
+
+const char *prof_time_resolution_mode_names[] = {
+	"default",
+	"high",
+};
+
+bool opt_high_resolution_timer =
```

Comment:
This should live in `jemalloc_preamble.h.in` and be `static const bool config_high_resolution_timer` ("opt" is for MALLOC_CONF-style configuration options; compiler / build time settings are under "config").